src/Grid.js

```diff
@@ -165,7 +169,7 @@ Grid.prototype.redraw = function(newOptions, callback) {
   var items;
 
   window.requestAnimationFrame(function() {
-    if (this.columns !== newOptions.columns) {
+    if (this.columns !== newOptions.columns || this.columnClasses !== newOptions.columnClasses) {
```

Comment:
I don't think this is needed here. Is there a use case where columns need to be redrawn to the same number of columns as before?